src/test/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImplTest.java

```diff
@@ -455,21 +492,70 @@ void positiveAnonymiseEventByIds() {
         doReturn(event3).when(eventService).getEventByEveId(3);
         doReturn(event1).when(eventService).getEventByEveId(4);
         UserAccountEntity userAccount = mock(UserAccountEntity.class);
-        dataAnonymisationService.anonymiseEventByIds(userAccount, List.of(1, 2, 3, 4));
+        doNothing().when(dataAnonymisationService).registerDataAnonymisation(any(), any(EventEntity.class), anyBoolean());
+
+        dataAnonymisationService.anonymiseEventByIds(userAccount, List.of(1, 2, 3, 4), isManuallyRequested);
 
 
-        verify(dataAnonymisationService, times(1)).anonymiseEventEntity(userAccount, event1, false);
-        verify(dataAnonymisationService, times(1)).anonymiseEventEntity(userAccount, event2, false);
-        verify(dataAnonymisationService, times(1)).anonymiseEventEntity(userAccount, event3, false);
+        verify(dataAnonymisationService).anonymiseEventEntity(userAccount, event1, false, isManuallyRequested);
+        verify(dataAnonymisationService).anonymiseEventEntity(userAccount, event2, false, isManuallyRequested);
+        verify(dataAnonymisationService).anonymiseEventEntity(userAccount, event3, false, isManuallyRequested);
 
-        verify(eventService, times(1)).getEventByEveId(1);
-        verify(eventService, times(1)).getEventByEveId(2);
-        verify(eventService, times(1)).getEventByEveId(3);
-        verify(eventService, times(1)).getEventByEveId(4);
+        verify(eventService).getEventByEveId(1);
+        verify(eventService).getEventByEveId(2);
+        verify(eventService).getEventByEveId(3);
+        verify(eventService).getEventByEveId(4);
 
-        verify(eventService, times(1)).saveEvent(event1);
-        verify(eventService, times(1)).saveEvent(event2);
-        verify(eventService, times(1)).saveEvent(event3);
+        verify(eventService).saveEvent(event1);
+        verify(eventService).saveEvent(event2);
+        verify(eventService).saveEvent(event3);
         verifyNoMoreInteractions(eventService);
+        verify(dataAnonymisationService).registerDataAnonymisation(userAccount, event1, isManuallyRequested);
+        verify(dataAnonymisationService).registerDataAnonymisation(userAccount, event2, isManuallyRequested);
+        verify(dataAnonymisationService).registerDataAnonymisation(userAccount, event3, isManuallyRequested);
+    }
+
+    @ParameterizedTest(name = "Register data anonymisation event with isManuallyRequested = {0}")
+    @ValueSource(booleans = {true, false})
+    void registerDataAnonymisationEvent(boolean isManuallyRequested) {
```

Comment:
Can you amend the name to use the new convention please

Reply:
Updated